spec.html

```diff
@@ -38589,6 +38768,281 @@ <h1>JSON [ @@toStringTag ]</h1>
   </emu-clause>
 </emu-clause>
 
+<emu-clause id="sec-managing-memory">
+  <h1>Managing Memory</h1>
```

Comment:
(No action needed for this PR)

To the other editors @bakkot @ljharb @michaelficarra: we should have links to WeakMap and WeakSet in this new section.